yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -74,10 +74,51 @@ var assessMigrationCmd = &cobra.Command{
 	},
 }
 
+func registerSourceDBConnFlagsForAM(cmd *cobra.Command) {
+	cmd.Flags().StringVar(&source.DBType, "source-db-type", "postgresql",
+		"source database type: (postgresql)\n")
+
+	cmd.Flags().StringVar(&source.Host, "source-db-host", "localhost",
+		"source database server host")
+
+	cmd.Flags().IntVar(&source.Port, "source-db-port", 0,
+		"source database server port number. Default: PostgreSQL(5432)")
+
+	cmd.Flags().StringVar(&source.User, "source-db-user", "",
+		"connect to source database as the specified user")
+
+	// TODO: All sensitive parameters can be taken from the environment variable
+	cmd.Flags().StringVar(&source.Password, "source-db-password", "",
+		"source password to connect as the specified user. Alternatively, you can also specify the password by setting the environment variable SOURCE_DB_PASSWORD. If you don't provide a password via the CLI, yb-voyager will prompt you at runtime for a password. If the password contains special characters that are interpreted by the shell (for example, # and $), enclose the password in single quotes.")
+
+	cmd.Flags().StringVar(&source.DBName, "source-db-name", "",
+		"source database name to be migrated to YugabyteDB")
+
+	cmd.Flags().StringVar(&source.Schema, "source-db-schema", "",
+		"source schema name(s) to export\n"+
+			`Note: in case of multiple schemas, use a comma separated list of schemas: "schema1,schema2,schema3"`)
```

Comment:
you can make use of `registerCommonSourceDBConnFlags()` for the above args

Reply:
+1. If it's only about the mentions of "oracle/mysql", see if it's possible to re-declare those flags or override their help msgs _after_ having called `registerSourceDBConnFlags` ?

Reply:
I think its better to keep it here as this is just for this command otherwise, I will have to add conditions in that function and duplicate the messaging for most of the flags.

Reply:
will try to do this refactoring thing in parallel to RC in a separate PR and merge this PR for the time being.